releasenotes/notes/qiskit_version-956916f7b8d7bbb9.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    The magic ``%qiskit_version_table`` from ``qiskit.tools.jupyter`` now includes all
+    the modules with ``'qiskit`` in the name.
```

Comment:
Maybe say "all imported packages with ``qiskit`` in the name"?

Reply:
reworded in bab5901ca463332e3d3224f0cefb5f4417b7a332